dapr-integration/README.md

```diff
@@ -0,0 +1,36 @@
+# Dapr Integrations 
+
+`idpBuilder` is extensible to launch custom Dapr patterns using package extensions. 
+
+Please use the below command to deploy an IDP reference implementation with an Argo application for preparing up the setup for terraform integrations:
+
+```bash
+idpbuilder create \
+  --use-path-routing \
+  --package-dir https://github.com/cnoe-io/stacks//ref-implementation \
```

Comment:
Please use the `-p` flag instead. The `--package-dir` flag is deprecated.

Reply:
done